concepts/strings/about.md

```diff
@@ -57,12 +57,6 @@ Some characters need to be [escaped][escape-characters] to be put in a string, e
 "\"A\" is the \#1st letter of the alphabet.\n"
 ```
 
-You can print a string to the console using [`IO.puts/2`][io-puts]:
-
-```elixir
-IO.puts("\"A\" is the \#1st letter of the alphabet.\n")
-```
-
```

Comment:
I think it's not necessary here, it's better if the students reads the `IO` concept if they want to know about printing strings.